### PR TITLE
Add support articles

### PR DESCRIPTION
**Pull request by samajammin**
https://www.notion.so/efdn/Pull-education-content-into-the-product-52c7cc72068b42d7808449a3bc5e7838

- Nest info articles under /about route
- Refactor `BackToProjects` to more flexible `BackLink` component
- Add additional articles from Ezra

Full list of pages updated/added:
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/how-it-works
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/how-it-works/contributors
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/how-it-works/recipients
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/quadratic-funding
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/public-goods
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/decentralization
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/sybil-resistance
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/maci
- https://deploy-preview-374--eth2clrfund.netlify.app/#/about/layer-2

### CODE REVIEW

File: vue-app/src/views/About.vue [hunk 1]
```diff
@@ -26,19 +28,63 @@
...
+          >Read more on clr.fund's decentralization</links
+        >.
+      </li>
+      <li>
+        Scalability — Because using Ethereum can be expensive for users,
+        clr.fund uses "layer 2" supplementary networks to get most of the
+        benefits of the main Ethereum network but with faster, cheaper
+        transactions.
+        <links to="/about/layer-2"
+          >Learn more about clr.fund's user of layer 2.</links
```
Comment (corwintines) [inline vue-app/src/views/About.vue]:
```suggestion
          >Learn more about clr.fund's use of layer 2.</links
```

File: vue-app/src/views/AboutDecentralization.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        the funds from and encrypted information about contributions to
+        individual projects
+      </li>
+      <li>
+        the mechanism for distributing funds once a round's results are
+        finalized
+      </li>
+    </ul>
+    <p>
+      Each of these pieces exists in an Ethereum smart contract (see the code
```
Comment (corwintines) [inline vue-app/src/views/AboutDecentralization.vue:39]: Should this `(see the code here)` section link out to the repo?

File: vue-app/src/views/AboutLayer2.vue [hunk 3]
```diff
@@ -1,43 +1,90 @@
...
 
-    <h2>Layer 2</h2>
+    <h2>Clr.fund on Layer 2</h2>
     <p>
-      <b
-        >tl;dr: if you want to create loads of accounts to vote for your
-        favourite projects multiple times, you're ngmi (not going to make it)</b
-      >
+      <b>
+        tl;dr: clr.fund runds on
```
Comment (corwintines) [inline vue-app/src/views/AboutLayer2.vue]:
```suggestion
        tl;dr: clr.fund runs on
```

File: vue-app/src/views/AboutLayer2.vue [hunk 4]
```diff
@@ -1,43 +1,90 @@
...
-      contributing to your favourite projects.
+      So, the increasing cost of using Ethereum demonstrates that it's useful,
+      which is great, but it also presents a problem for end users who don't
+      want to pay fees in the 10s or 100s of dollars.
     </p>
 
-    <h2>Why Layer2?</h2>
+    <h2>Layer 2s for Scalability</h2>
+    <p>
+      The main Ethereum blockchain, "layer 1," may be upgraded to reduce costs
```
Comment (corwintines) [inline vue-app/src/views/AboutLayer2.vue]:
```suggestion
      The main Ethereum blockchain, "layer 1", may be upgraded to reduce costs
```

File: vue-app/src/views/AboutLayer2.vue [hunk 5]
```diff
@@ -1,43 +1,90 @@
...
+      in the future (this is one of the goals of Eth2, which this clr.fund round
+      is helping realize!).
+    </p>
+    <p>
+      In the immediate term, though, "layer 2" solutions are already helping
+      dramatically reduce costs. Most layer 2s are "rollups," blockchain-esque
```
Comment (corwintines) [inline vue-app/src/views/AboutLayer2.vue]:
```suggestion
      dramatically reduce costs. Most layer 2s are "rollups", blockchain-esque
```

File: vue-app/src/views/AboutMaci.vue [hunk 6]
```diff
@@ -32,34 +109,92 @@
...
         There are limits on the number of projects, contributors, and
-        contributions. This means you may miss out but know that it's for the
-        integrity of the round.
+        contributions. This means you may miss out if a round hits capacity but
+        know that it's for the integrity of the round.
       </li>
       <li>
-        You can only contribute one amount per round. Once you have contributed,
-        you can add/remove projects but you can't increase your total amount.
+        You can only contribute once amount per round. Once you have
```
Comment (corwintines) [inline vue-app/src/views/AboutMaci.vue]:
```suggestion
        You can only contribute once per round. Once you have
```

File: vue-app/src/views/AboutMaci.vue [hunk 7]
```diff
@@ -32,34 +109,92 @@
...
+        contributed, you can add/remove projects and reallocate your funds but
+        you can't increase your total contributionamount.
```
Comment (corwintines) [inline vue-app/src/views/AboutMaci.vue]:
```suggestion
        you can't increase your total contribution amount.
```

**Review by corwintines — COMMENTED** [review]
@samajammin just left a few comments, and fixed a couple li styles. Saw you had some TODO's around the token symbol and chain label, so I added a commit to address those as well. I think this is good to go after you take a look at those comments.

**Comment by corwintines** [comment]
I am curious if there is something we can do to improve the li style, looks a bit crammed with the content heavy ones. But I think we can address that in a different issue.

File: vue-app/src/views/AboutDecentralization.vue [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+      here), meaning its functionality is maintained by Ethereum miners, a
```
Comment (samajammin) [inline vue-app/src/views/AboutDecentralization.vue]:
```suggestion
      <links to="https://github.com/clrfund/monorepo">here</links>), meaning its functionality is maintained by Ethereum miners, a
```

**Comment by samajammin** [comment]
> I am curious if there is something we can do to improve the li style, looks a bit crammed with the content heavy ones. But I think we can address that in a different issue.

@corwintines agreed - I have a separate task to fix styling: https://www.notion.so/efdn/Style-all-about-content-e72a47edc62e40c88724c5441ce23038

**Pull request merged**